common/aicore/cm.h

```diff
@@ -22,6 +22,22 @@ struct cm_parameter {
 
   int factor[O_LAST];
   int happy_factor;
+
+  // Copy assignment operator
```

Comment:
Docstrings should start with three slashes (`///`) or `/**` because that how Doxygen "sees" them (this is very common for documentation generators, with a long history dating back to at least 1995 and the first release of Java).